lib/tasks/translatable.rake

```diff
@@ -198,14 +201,19 @@ namespace :translatable do
     return hash
   end
 
-  def files_to_translate
-    Dir.glob("{app,lib,config,locale}/**/*.{rb,erb,md,haml,slim,rhtml}")
+  # TODO: exclude app/views/branded
```

Comment:
I had forgotten that we set the site up to look for branded versions of files there 😆 
Will need to move my files from my 'dmptool' subdirs to 'branded'
We should also consider doing the same in the JS dir